imageio/v2.py

```diff
@@ -250,7 +250,7 @@ def is_volume(ndimage):
 # Base functions that return a reader/writer
 
 
-def get_reader(uri, format=None, mode="?", **kwargs):
+def get_reader(uri, format=None, mode="?", pilmode=None, **kwargs):
```

Comment:
```suggestion
def get_reader(uri, format=None, mode="?", **kwargs):
```

Moving the renaming to `LegacyReader` will avoid us bleeding a plugin-specific kwarg into the high-level API. I'd very much prefer that, because it will leave the API as-is :)